opencog/nlp/chatbot-psi/actions.scm

```diff
@@ -72,8 +72,9 @@
 (define (say . words)
     (define utterance (string-join (map cog-name words)))
 
-    ; Remove those '[' and ']' that may exist in the output
-    (set! utterance (string-filter (lambda (c) (not (or (char=? #\[ c) (char=? #\] c)))) utterance))
+    ; Remove those '[', ']' and '\' that may exist in the output
```

Comment:
I hit one reply that has a WordNode `(WordNode "\"hello\".")`

I guess '\' may be used to escape quotes exist in some word in some aiml rule, and after converting to opencog psi rules they get the chance to be part of a word